src/buildMathML.js

```diff
@@ -325,6 +325,10 @@ groupTypes.op = function(group) {
         // This is a symbol. Just add the symbol.
         node = new mathMLTree.MathNode(
             "mo", [makeText(group.value.body, group.mode)]);
+    } else if (group.value.value) {
+        // do nothing for now
+        var inner = buildExpression(group.value.value, options);
+        node = new mathMLTree.MathNode("mo", inner);
```

Comment:
I think this makes sense.  Is there something additional that you think we should be doing in the future?